src/Accelerators/NNPA/Runtime/zDNNExtension/zDNNExtension.h

```diff
@@ -121,6 +121,17 @@ void zDNNExtensionInit();
 // Helper Functions
 // -----------------------------------------------------------------------------
 
+inline void omUnreachable() {
```

Comment:
Very good. The name prefix `om` made me think of 2 things: 
1) If you see this as useful also for our `cruntime`, then let's move it there. Note that we should be able to do that later too. 
2) Do we need a prefix for the entry points into this new library? We can argue that we do not, as it is only used by our compiler and is technically not exposed to the outside.

I don't have a firm opinion on either points.

Reply:
Thanks for your comments! Both two things look reasonable to me. I see that cruntime is now using `assert` instead of `unreachable` and there is no a common place for shared functions. Another PR for that seems better.